### ISSUE TITLE
NIC offloading needs to be disabled at every boot

### ISSUE THREAD

**Issue by dougburks** [Security-Onion-Solutions/securityonion-saltstack#319]
It looks like NIC offloading is only disabled once when setup runs.  When the box reboots, it reverts back to default NIC settings with offloading enabled.  We need to disable NIC offloading at every boot.

As mentioned in https://github.com/Security-Onion-Solutions/securityonion-saltstack/issues/62, our current platform does the disabling in `/etc/network/interfaces` so that it gets applied at every boot.

**Comment by dougburks** [comment]
I do see that `/etc/NetworkManager/dispatcher.d/00-so-checksum-offload-disable` got created as follows:
```
#!/bin/bash

if [ "$NM_DISPATCHER_ACTION" == "pre-up" ]; then
    if ["$DEVICE_IFACE" !== "ens33"]; then
        for i in rx tx sg tso ufo gso gro lro; do 
            ethtool -K $DEVICE_IFACE $i off; 
        done
    fi
fi
```
This VM has 2 interfaces, ens33 and ens34.  ens33 is the management interface and is correctly excluded in the file above.  ens34 is the sniffing interface and it needs offloading disabled, but here's what it shows:
```
ethtool -k ens34 |grep offload
tcp-segmentation-offload: on
udp-fragmentation-offload: off [fixed]
generic-segmentation-offload: on
generic-receive-offload: on
large-receive-offload: off [fixed]
rx-vlan-offload: on
tx-vlan-offload: on [fixed]
l2-fwd-offload: off [fixed]
hw-tc-offload: off [fixed]
rx-udp_tunnel-port-offload: off [fixed]
```

**Comment by rwwiv** [comment]
This was resolved with #320, with further non-breaking changes in #321.
